### ISSUE TITLE
Update PATCH Request Doesn't Work in Development

### ISSUE THREAD

**Issue by aviflombaum** [learn-co-curriculum/sinatra-pigeon#15]
Track: Learn.co Web Development with Ruby on Rails
Batch: 001

I can't get this route to trigger: https://github.com/learn-co-curriculum/sinatra-pigeon/blob/solution/app/controllers/pigeons_controller.rb#L32

It's for the `PATCH /pigeons/:id`. The form uses the [`_method` correctly](https://github.com/learn-co-curriculum/sinatra-pigeon/blob/solution/app/controllers/pigeons_controller.rb#L32), but I just can't get it to work.

![Error](https://dl.dropboxusercontent.com/s/u0qvzwj0q4tg9fn/2015-08-29%20at%203.11%20PM.png)

The test suite does pass correctly.

```
  update: PATCH /pigeons/id
    redirects to the pigeons show page
```

Not sure, there's just something weird going on here...

Steps to Recreate:

1. Clone
2. Checkout Solution
3. Run `rake db:migrate` and then `rake db:migrate SINATRA_ENV=test`
4. Shotgun
5. Navigate to http://127.0.0.1:9393/pigeons/new
6. Make a new pigeon
7. Hit edit.
8. Update the pigeon.
9. Submit the update form.
10. You should see sinatra complain about the route.

You will see that when you run `rspec` though the test suite works.

**Comment by SophieDeBenedetto** [comment]
closing this issue. I just spoke to Garett and the problem was that he didn't add the `Rack::MethodOverrid` middleware to his `config.ru`. A note to do so is in the instructions for this lab.

**Comment by aviflombaum** [comment]
Closed by 341d3fa17923c95d70275de8630020bb03b0dbf3
